devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy

```diff
@@ -99,6 +99,7 @@ def parse(String description) {
 		} else {
 			Map descMap = zigbee.parseDescriptionAsMap(description)
 			if (descMap?.clusterInt == 0x0001 && descMap.commandInt != 0x07 && descMap?.value) {
+				log.info "BATT METRICS - ${descMap?.attrInt} attr; value: ${descMap?.value}, dec value: ${Integer.parseInt(descMap.value, 16)}, curr percent: ${device.currentState("battery")?.value}, device: ${device.getDataValue("manufacturer")} ${device.getDataValue("model")}"
```

Comment:
I think it would be clearer to put `attr:` before the attribute value

Reply:
@tpmanley I tweaked it. And apparently it means I dismissed your approval. Anyway, once I get it again I'll merge it.